platform/Sleep.roc

```diff
@@ -1,13 +0,0 @@
-module [
```

Comment:
Why are we removing `Sleep` and the related example? Is this temporary for getting everything working? It seems to be in `basic-cli`, so maybe it got missed in the great copy of `87?

Reply:
The way it was implemented it was putting the whole thread to sleep. I think I added it mindlessly porting across from basic-cli... but it's not something I think we want in a multithreaded HTTP web server, at least not like this.

Reply:
basic webserver spawns a thread per request. So sleep is actually fine (though silly to do).